hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/HawkbitUI.java

```diff
@@ -201,6 +204,12 @@ public void afterViewChange(final ViewChangeEvent event) {
         LOG.info("Current locale of the application is : {}", i18n.getLocale());
     }
 
+    private void closeAllWindows(final LayoutClickEvent event) {
+        for (final Window window : getWindows()) {
+            window.close();
```

Comment:
As long i remember this will not work for every browser  e.g. Safari.
I think is better to use `UI#removeWindow(window);`